build-scripts/k8s_releases.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
```

Comment:
I don't think we need to add all of this here (see my comment above).

Reply:
I'm not sure I understand what you mean.

Reply:
Oh, I see. You mean moving the pre-release checks to the "update-components.yaml" script and the branch management to the other project.

Reply:
One problem is that ``update-components.yaml`` will simply create a PR that updaets the ``component`` versions. However, we also need to sync the changes from the ``main`` branch of k8s-snap, which is why the new job currently does a force push.